### PR TITLE
chore: Added pipeline to open PR on homebrew-core project

### PR DESCRIPTION
**Pull request by juliosaraiva**

### CODE REVIEW

File: .github/workflows/brew-publish.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  test:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Update Homebrew formulae
+        uses: dawidd6/action-homebrew-bump-formula@v3
+        with:
+          org: pyroscope
+          token: ${{ secrets.TOKEN }}
```
Comment (petethepig) [inline .github/workflows/brew-publish.yml:14]: Can you call it something else, like `secrets.GITHUB_HOMEBREW_TOKEN`? I just want to make sure that when we have a bunch of secrets in the repo it's still clear what each token does.

Comment (petethepig) [reply .github/workflows/brew-publish.yml:14]:
```suggestion
          token: ${{ secrets. GITHUB_HOMEBREW_TOKEN }}
```

Comment (juliosaraiva) [reply .github/workflows/brew-publish.yml]: LGTM

**Review by petethepig — APPROVED** [review]
Thank you @juliosaraiva! I'm going to test it out next time we release!

**Pull request merged**